test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -303,13 +303,6 @@ def host_setup_for_tensorflow_inference(
     is_graviton=False,
     model_name=None,
 ):
-    # Wait for any existing apt-get calls to finish before moving on
-    # TODO(Mike Schneider): Improve this by adding a check for running apt-get processes and wait for them to finish,
-    # then timeout after a given amount of time if other apt-get calls are taking too long.
-    ec2_connection.run((f"sleep 180"), hide=True)
-
-    # Install PIP so we can test
-    ec2_connection.run((f"sudo apt-get update && sudo apt-get install -y python3-pip"), hide=True)
 
```

Comment:
Having deleted all the lines of code above, this line is an empty line at the start of the function, causing the formatting check failure. Will fix this in the next revision.